bibliopixel/main/main.py

```diff
@@ -5,7 +5,8 @@
 
 __all__ = ['main']
 
-COMMANDS = ('all_pixel', 'devices', 'demo', 'run', 'set', 'show')
+COMMANDS = 'all_pixel', 'devices', 'demo', 'run', 'set', 'show',
```

Comment:
Urg, sorry, I know you prefer the parens.  Is this the only change?  I'll revert that.  Moment pls...